data.json

```
@@ -2208,6 +2208,15 @@
                 "JavaScript"
             ],
             "description": "a react-based framework that provides accessibility, modularity, responsiveness, and theming in a tidy package"
-        }
+        },
+        {
+            "name": "Todo List App",
+            "link": "https://github.com/pixelledbot/todo-list-app",
```

```suggestion
            "link": "https://github.com/pixelledbot/to-do-list",
```

Did you mean this?

Otherwise, check if your repository is private, that may be causing the issue of the link being broken.

---

yes, I meant that